lgn-provers/src/provers/v1/query/euclid_prover.rs

```diff
@@ -92,12 +93,16 @@ impl StorageQueryProver for EuclidQueryProver {
         let input = QueryCircuitInput::Query(circuit_input);
         let proof = self.params.generate_proof(input)?;
 
+        let proof_type = "universal circuit";
```

Comment:
We should replace the space with an underline:

```suggestion
        let proof_type = "universal_circuit";
```

I believe that spaces are not supported for labels by prometheus:

> Labels may contain ASCII letters, numbers, as well as underscores. They must match the regex [a-zA-Z_][a-zA-Z0-9_]*.

Reply:
Oh yeah they are not. well spotted

Reply:
Fixed